src/main/java/org/broadinstitute/hellbender/utils/fermi/FermiLiteAssembler.java

```diff
@@ -15,35 +15,89 @@
+    public int getCleaningFlag() { return opts.getInt(24); }
+    public void setCleaningFlag( final int cleaningFlag ) { opts.putInt(24,cleaningFlag); }
+    public int getCleaningMinOverlap() { return opts.getInt(28); }
+    public void setCleaningMinOverlap( final int minOverlap ) { opts.putInt(28,minOverlap); }
+    public int getCleaningELen() { return opts.getInt(32); }
+    public void setCleaningELen( final int eLen ) { opts.putInt(32,eLen); }
+    public int getCleaningMinEnsr() { return opts.getInt(36); }
+    public void setCleaningMinEnsr( final int minEnsr ) { opts.putInt(36,minEnsr); }
+    public int getCleaningMinInsr() { return opts.getInt(40); }
+    public void setCleaningMinInsr( final int minInsr ) { opts.putInt(40,minInsr); }
+    public int getCleaningMaxBDist() { return opts.getInt(44); }
+    public void setCleaningMaxBDist( final int maxBDist ) { opts.putInt(44,maxBDist); }
+    public int getCleaningMaxBDiff() { return opts.getInt(48); }
+    public void setCleaningMaxBDiff( final int maxBDiff ) { opts.putInt(48,maxBDiff); }
+    public int getCleaningMaxBVtx() { return opts.getInt(52); }
+    public void setCleaningMaxBVtx( final int maxBVtx ) { opts.putInt(52,maxBVtx); }
+    public int getCleaningMinMergeLen() { return opts.getInt(56); }
+    public void setCleaningMinMergeLen( final int minMergeLen ) { opts.putInt(56,minMergeLen); }
+    public int getCleaningTrimLen() { return opts.getInt(60); }
+    public void setCleaningTrimLen( final int trimLen ) { opts.putInt(60,trimLen); }
+    public int getCleaningTrimDepth() { return opts.getInt(64); }
+    public void setCleaningTrimDepth( final int trimDepth ) { opts.putInt(64,trimDepth); }
+    public float getCleaningDRatio1() { return opts.getFloat(68); }
+    public void setCleaningDRatio1( final float dRatio1 ) { opts.putFloat(68,dRatio1); }
+    public float getCleaningMaxBCov() { return opts.getFloat(72); }
+    public void setCleaningMaxBCov( final float maxBCov ) { opts.putFloat(72,maxBCov); }
+    public float getCleaningMaxBFrac() { return opts.getFloat(76); }
+    public void setCleaningMaxBFrac( final float maxBFrac ) { opts.putFloat(76,maxBFrac); }
+
     /**
      * Create an assembly from a collection of objects that implement BasesAndQuals.
      */
-    public FermiLiteAssembly createAssembly( final Iterable<BasesAndQuals> basesAndQuals ) {
+    public FermiLiteAssembly createAssembly( final Iterable<? extends BasesAndQuals> basesAndQuals ) {
         return createAssembly(basesAndQuals, bAndQ -> bAndQ);
     }
 
     /**
      * Create an assembly from a collection of objects that can be transformed (with a lambda) into BasesAndQuals.
      */
     public <T> FermiLiteAssembly createAssembly( final Iterable<T> reads, final Function<T,BasesAndQuals> func ) {
-        final ByteBuffer assemblyData = createAssemblyData(makeReadData(reads, func));
+        final ByteBuffer assemblyData = createAssemblyData(opts,makeReadData(reads, func));
```

Comment:
What does / should happen if someone calls close on the assembler and then calls createAssembly?  It seems like there might need to be a guard to prevent that.  Similarly, what happens if you get/modify the opts after closing?